openlibrary/templates/account/password/reset_success.html

```diff
@@ -7,5 +7,5 @@ <h1>$_("Hi, %(user)s", user=username)!</h1>
 </div>
 
 <div id="contentBody">
-    <p>$:_("Your password has been updated. Please <a href='/account/login?redirect=/'>log in to continue</a>.")</p>
+    <p>$:_("Your password has been updated. Please <a href="/account/login?redirect=/">log in to continue</a>.")</p>
```

Comment:
CAUTION: Outer string vs. inner strings.

Reply:
Fixed this  in the latest PR